applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_axisymmetric_integration_coefficients.cpp

```diff
@@ -0,0 +1,62 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//                   Marjan Fathian
+//                   Gennady Markelov
+//
+
+#include "custom_elements/axisymmetric_integration_coefficients.h"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+#include "tests/cpp_tests/test_utilities/element_setup_utilities.h"
+#include "tests/cpp_tests/test_utilities/model_setup_utilities.h"
+
+#include <boost/numeric/ublas/assignment.hpp>
+
+using namespace Kratos;
+using namespace std::string_literals;
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(AxisymmetricIntegrationCoefficients_ReturnsCorrectValue, KratosGeoMechanicsFastSuiteWithoutKernel)
```

Comment:
Also here, the naming can be a bit more consistent with the tested class 
```suggestion
KRATOS_TEST_CASE_IN_SUITE(IntegrationCoefficientModifierForAxisymmetricElement_ReturnsCorrectValue, KratosGeoMechanicsFastSuiteWithoutKernel)
```

Reply:
named as `AxisymmetricIntegrationCoefficientsCalculator`. Similar changes for other tests